notebook/static-src/edit/js/menubar.js

```diff
@@ -1,17 +1,14 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
-
-define([
-    'jquery',
-    'base/js/namespace',
-    'base/js/utils',
-    'base/js/dialog',
-    'codemirror/lib/codemirror',
-    'codemirror/mode/meta',
-    'bootstrap',
-], function($, IPython, utils, dialog, CodeMirror) {
     "use strict";
-    
+
+    var IPython = require('base/js/namespace');
+    var utils = require('base/js/utils');
+    var dialog = require('base/js/dialog');
+    var CodeMirror = require('codemirror/lib/codemirror');
+    require('codemirror/mode/meta');
+    requirejs(['bootstrap'], function() {});
```

Comment:
fudge!  The bootstrap one shouldn't be there, globals handles that now.  The codemirror one is supposed to inject into the codemirror namespace I think...  That may be a source of the problems though.  I may end up loading all of codemirror in the globals file.